Makefile

```diff
@@ -49,45 +125,32 @@ check-format: vendor/.up-to-date
 .PHONY: goimports go-fmt format-code
 # Format the code using goimports.  Depends on the vendor directory because goimports needs
 # to be able to resolve the imports.
-goimports go-fmt format-code: vendor/.up-to-date
+goimports go-fmt format-code fix: vendor/.up-to-date
 	$(DOCKER_GO_BUILD) goimports -w lib
 
-.PHONY: update-vendor
-# Update the pins in glide.lock to reflect the updated glide.yaml.
-# Note: no dependency on glide.yaml so we don't automatically update glide.lock without
-# developer intervention.
-update-vendor glide.lock:
-	$(DOCKER_GO_BUILD) glide up --strip-vendor
-	touch vendor/.up-to-date
-	# Optimization: since glide up does the job of glide install, flag to the
-	# vendor target that it doesn't need to do anything.
-	$(eval VENDOR_REMADE := true)
-
-.PHONY: vendor
-# Rebuild the vendor directory.
-vendor vendor/.up-to-date: glide.lock
-	# To update upstream dependencies, delete the glide.lock file first
-	# or run make update-vendor.
-	# To build without Docker just run "glide install -strip-vendor"
-	if ! $(VENDOR_REMADE); then \
-	    $(DOCKER_GO_BUILD) glide install --strip-vendor && \
-	    touch vendor/.up-to-date; \
-	fi
-
-.PHONY: ut
-## Run the UTs locally.  This requires a local etcd and local kubernetes master to be running.
-ut: vendor/.up-to-date
+.PHONY: install-git-hooks
+## Install Git hooks
+install-git-hooks:
+	./install-git-hooks
+
+###############################################################################
+# Tests
+###############################################################################
+.PHONY: ut-native
+## Run the UTs natively.  This requires a local etcd and local kubernetes master to be running.
+ut-native: vendor/.up-to-date
 	./run-uts
 
-.PHONY: test-containerized
+.PHONY:ut
 ## Run the tests in a container. Useful for CI, Mac dev.
-test-containerized: vendor/.up-to-date run-etcd run-kubernetes-master
+ut: vendor/.up-to-date run-etcd run-kubernetes-master
```

Comment:
This makes me indescribably happy.

Reply:
Admittedly, I don't like `vendor/.up-to-date` as a target, since it means a manually performed `glide up` will get overwritten.

Might just have to be a live with... but it's a bit painful.

Reply:
Got rid if the .up-to-date business